### ISSUE TITLE
20190523 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#77]
# 20190523 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [cloudkj/scar](https://github.com/cloudkj/scar) （Deploy static websites in seconds - with HTTPS, a global CDN, and custom domains.）
* [PavelDoGreat/WebGL-Fluid-Simulation](https://github.com/PavelDoGreat/WebGL-Fluid-Simulation) （Play with fluids in your browser (works even on mobile)）
* [TankerHQ/sdk-js](https://github.com/TankerHQ/sdk-js) （Tanker client-side encryption SDK for JavaScript）
* [azl397985856/leetcode](https://github.com/azl397985856/leetcode) （leetcode题解，记录自己的leecode解题之路。）
* [developit/dlv](https://github.com/developit/dlv) （Safe deep property access in 120 bytes. x = dlv(obj, 'a.b.x')）
* [CyC2018/CS-Notes](https://github.com/CyC2018/CS-Notes) （? Computer Science Learning Notes）
* [imfunniee/gitfolio](https://github.com/imfunniee/gitfolio) （ personal website + blog for every github user）
* [agalwood/Motrix](https://github.com/agalwood/Motrix) （A full-featured download manger.）
* [sveltejs/svelte](https://github.com/sveltejs/svelte) （The magical disappearing UI framework）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [Binaryify/NeteaseCloudMusicApi](https://github.com/Binaryify/NeteaseCloudMusicApi) （        网易云音乐nodejs api      ）
* [robb0wen/tornis](https://github.com/robb0wen/tornis) （Tornis helps you watch and respond to changes in your browser's viewport &#x1f332;）
* [lib-pku/libpku](https://github.com/lib-pku/libpku) （北京大学课程资料整理）
* [transloadit/uppy](https://github.com/transloadit/uppy) （The next open source file uploader for web browsers ?）
* [axios/axios](https://github.com/axios/axios) （Promise based HTTP client for the browser and node.js）
* [syhyz1990/baiduyun](https://github.com/syhyz1990/baiduyun) （油猴脚本 直接下载百度网盘和百度网盘分享的文件,直链下载超级加速）
* [testing-library/react-testing-library](https://github.com/testing-library/react-testing-library) （&#x1f410; Simple and complete React DOM testing utilities that encourage good testing practices.）
* [facebook/react-native](https://github.com/facebook/react) （        A framework for building native apps with React.      ）
* [tailwindcss/custom-forms](https://github.com/tailwindcss/custom-forms) （A better base for styling form elements with Tailwind CSS.）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [mrdoob/three.js](https://github.com/mrdoob/three.js) （        JavaScript 3D library.      ）
* [weilanwl/ColorUI](https://github.com/weilanwl/ColorUI) （鲜亮的高饱和色彩，专注视觉的小程序组件库）
* [iview/iview-admin](https://github.com/iview/iview-admin) （Vue 2.0 admin management system template based on iView）
* [Molunerfinn/PicGo](https://github.com/Molunerfinn/PicGo) （A simple &amp; beautiful tool for pictures uploading built by electron-vue）

## Vue
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [jolaleye/cssfx](https://github.com/jolaleye/cssfx) （✨ Beautifully simple click-to-copy CSS effects）
* [dcloudio/uni-app](https://github.com/dcloudio/uni-app) （使用 Vue.js 开发跨平台应用的前端框架）
* [lenve/vhr](https://github.com/lenve/vhr) （微人事是一个前后端分离的人力资源管理系统，项目采用SpringBoot+Vue开发。）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [lenve/VBlog](https://github.com/lenve/VBlog) （V部落，Vue+SpringBoot实现的多用户博客管理平台!）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [sendya/ant-design-pro-vue](https://github.com/sendya/ant-design-pro-vue) （??‍???‍? Use Ant Design Vue like a Pro! Demo: https://pro.loacg.com | React: http://pro.ant.design/）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [ljphilp/vue-element-super](https://github.com/ljphilp/vue-element-super) （vue后台管理系统框架，快速开发）
* [neuronetio/gantt-elastic](https://github.com/neuronetio/gantt-elastic) （Gantt Chart [ javascript gantt chart, gantt component, vue gantt, vue gantt chart, responsive gantt, project manager , vue projects ]）
* [tikazyq/crawlab](https://github.com/tikazyq/crawlab) （Celery-based web crawler admin platform for managing distributed web spiders regardless of languages and frameworks.）
* [epicmaxco/vuestic-admin](https://github.com/epicmaxco/vuestic-admin) （Vue.js admin dashboard）
* [god-of-front-end/shop_template](https://github.com/god-of-front-end/shop_template) （电商模板）
* [wuyouzhuguli/FEBS-Vue](https://github.com/wuyouzhuguli/FEBS-Vue) （SpringBoot，Shiro，JWT，Vue &amp; Ant Design 前后端分离权限管理系统，预览地址（小油管，初次加载较慢）：）
* [2017coding/BBS_admin](https://github.com/2017coding/BBS_admin) （vue后台管理框架，vue+ElementUI+axios 接入数据，模仿segmentfault，做相关的后台管理系统 https://lyh.red/admin/）
* [Akryum/vue-virtual-scroller](https://github.com/Akryum/vue-virtual-scroller) （Smooth scroll with any amount of data）
* [vueComponent/ant-design-vue-pro](https://github.com/vueComponent/ant-design-vue-pro) （An enterprise-class UI components based on Ant Design and Vue. &#x1f41c;）
* [DCzajkowski/vue-pure-lightbox](https://github.com/DCzajkowski/vue-pure-lightbox) （Very simple lightbox plugin (without any dependencies) for Vuejs &#x1f305;）
* [jdf2e/nutui](https://github.com/jdf2e/nutui) （一个基于Vue2.0的轻量级移动端组件库）

## HTML
* [jaywcjlove/linux-command](https://github.com/jaywcjlove/linux-command) （        Linux命令大全搜索工具，内容包含Linux命令手册、详解、学习、搜集。      ）
* [jaywcjlove/awesome-mac](https://github.com/jaywcjlove/awesome-mac) （         This repo is a collection of awesome Mac applications and tools for developers and designers.      ）
* [flutterchina/flutter-in-action](https://github.com/flutterchina/flutter-in-action) （《Flutter实战》电子书）
* [ColorlibHQ/AdminLTE](https://github.com/ColorlibHQ/AdminLTE) （AdminLTE - Free Premium Admin control Panel Theme Based On Bootstrap 3.x）
* [fguby/Electron-elf](https://github.com/fguby/Electron-elf) （使用electron和live2D开发的类似桌面精灵的应用（A desktop application developed using electron and live2D））
* [facebookresearch/fastText](https://github.com/facebookresearch/fastText) （Library for fast text representation and classification.）
* [awslabs/aws-well-architected-labs](https://github.com/awslabs/aws-well-architected-labs) （Hands on labs and code to help you learn, measure, and build using architectural best practices.）
* [phil-opp/blog_os](https://github.com/phil-opp/blog_os) （Writing an OS in Rust）
* [portainer/portainer](https://github.com/portainer/portainer) （        Simple management UI for Docker      ）
* [javascript-tutorial/en.javascript.info](https://github.com/javascript-tutorial/en.javascript.info) （Modern JavaScript Tutorial）
* [golang101/golang101](https://github.com/golang101/golang101) （Go语言101 : 一本侧重于Go语言语法和语义的编程解释和指导书）
* [usablica/intro.js](https://github.com/usablica/intro.js) （        A better way for new feature introduction and step-by-step users guide for your website and project.      ）
* [apachecn/scikit-learn-doc-zh](https://github.com/apachecn/scikit-learn-doc-zh) （&#x1f4d6; [译] scikit-learn（sklearn） 中文文档）
* [hunterlong/statping](https://github.com/hunterlong/statping) （Status Page for monitoring your websites and applications with beautiful graphs, analytics, and plugins. Run on any type of environment.）
* [yyq123/learn-vim](https://github.com/yyq123/learn-vim) （无废话极简版Vim学习笔记！文章按主题分拆为多个章节，并尽量控制每节的信息量；通过文字色彩和字体，将命令、快捷键突出显示；在每节结尾，提供一个命令列表，以便回顾文中介绍的重要命令。如果这些文章能对喜欢Vim的朋友有所益处，我将不胜荣幸。）
* [zjcqoo/zjcqoo.github.io](https://github.com/zjcqoo/zjcqoo.github.io) （test）
* [swagger-api/swagger-codegen](https://github.com/swagger-api/swagger-codegen) （        swagger-codegen contains a template-driven engine to generate documentation, API clients and server stubs in different languages by parsing your OpenAPI / Swagger definition.      ）
* [linuxacademy/devops-essentials-sample-app](https://github.com/linuxacademy/devops-essentials-sample-app) （Build amazing native and progressive web apps with open web technologies. One app running on everything &#x1f389;）
* [uikit/uikit](https://github.com/uikit/uikit) （A lightweight and modular front-end framework for developing fast and powerful web interfaces）
* [javascript-tutorial/ru.javascript.info](https://github.com/javascript-tutorial/ru.javascript.info) （Современный учебник JavaScript）

## CSS
* [twbs/bootstrap](https://github.com/twbs/bootstrap) （The most popular HTML, CSS, and JavaScript framework for developing responsive, mobile first projects on the web.      ）
* [tailwindcss/tailwindcss](https://github.com/tailwindcss/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [daneden/animate.css](https://github.com/daneden/animate.css) （        A cross-browser library of CSS animations. As easy to use as an easy thing.      ）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [Tencent/weui](https://github.com/Tencent/weui) （A UI library by WeChat official design team, includes the most useful widgets/modules in mobile web applications.）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [imfunniee/gitark](https://github.com/imfunniee/gitark) （ Dark theme for Github）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [troxler/awesome-css-frameworks](https://github.com/troxler/awesome-css-frameworks) （List of awesome CSS frameworks）
* [StylishThemes/GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark) （         Dark GitHub style      ）
* [carloscuesta/gitmoji](https://github.com/carloscuesta/gitmoji) （        An emoji guide for your commit messages. ?       ）
* [rstacruz/cheatsheets](https://github.com/rstacruz/cheatsheets) （My cheatsheets）
* [AnomalyInnovations/serverless-stack-com](https://github.com/AnomalyInnovations/serverless-stack-com) （        Source for Serverless-Stack.com      ）
* [IsmaelMartinez/teams-for-linux](https://github.com/IsmaelMartinez/teams-for-linux) （Unofficial Microsoft Teams for Linux client）
* [Microsoft/vscode-docs](https://github.com/Microsoft/vscode-docs) （Public documentation for Visual Studio Code）
* [miniMAC/magic](https://github.com/miniMAC/magic) （CSS3 Animations with special effects）

## TypeScript
* [Eugeny/terminus](https://github.com/Eugeny/terminus) （A terminal for a more modern age）

## Unknow
* [jlevy/the-art-of-command-line](https://github.com/jlevy/the-art-of-command-line) （        Master the command line, in one page      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [加我微信，进500人微信群，领100本前端电子书](http://t.cn/EV7pcM1)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
